### PR TITLE
PMM-2425 Time zones and time parameters

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: import-dashboards.py [hunk 1]
```diff
@@ -254,6 +257,18 @@ def set_home_dashboard(api_key):
...
+    print ' * Organization preferences: %r %r' % (r.status_code, r.content)
+    if r.status_code != 200:
+        # TODO sys.exit(-1)
+        return
+
+    res = json.loads(r.content)
+
+    data = json.dumps({'homeDashboardId': res['homeDashboardId'],'theme': res['theme'], 'timezone': "browser"})
+    r = requests.put('%s/api/org/preferences' % (HOST,), data=data, headers=grafana_headers(api_key))
+    print ' * Organization timezone preference set to browser: %r %r' % (r.status_code, r.content)
```
Comment (askomorokhov) [inline import-dashboards.py]: Please, surround top-level function and class definitions with two blank lines.

File: import-dashboards.py [hunk 2]
```diff
@@ -254,6 +257,18 @@ def set_home_dashboard(api_key):
     # cur.execute("REPLACE INTO preferences (id, org_id, user_id, version, home_dashboard_id, timezone, theme, created, updated) "
     #             "SELECT 1, 1, 0, 0, id, '', '', datetime('now'), datetime('now') from dashboard WHERE slug='home'")
 
+def set_org_timezone(api_key):
+    r = requests.get('%s/api/org/preferences' % (HOST,), headers=grafana_headers(api_key))
+    print ' * Organization preferences: %r %r' % (r.status_code, r.content)
+    if r.status_code != 200:
```
Comment (askomorokhov) [inline import-dashboards.py]: Please use `HTTPStatus.OK` from `http` lib.

File: import-dashboards.py [hunk 3]
```diff
@@ -254,6 +257,18 @@ def set_home_dashboard(api_key):
...
+        # TODO sys.exit(-1)
+        return
+
+    res = json.loads(r.content)
+
+    data = json.dumps({'homeDashboardId': res['homeDashboardId'],'theme': res['theme'], 'timezone': "browser"})
```
Comment (askomorokhov) [inline import-dashboards.py]: Please do not mix double and single quotes.

File: import-dashboards.py (hunk 2 above)
Comment (adivinho) [reply import-dashboards.py]: Looks like such lib has come up since the version 3.5

Comment (adivinho) [reply import-dashboards.py]: I have replaced it by the OK constant from httplib

**Comment by atymchuk** [comment]
Closing this as stale. Please re-open if needed.

**Pull request closed without merging**